doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,845 @@
+  * New application specific Kconfig option to enable :ref:`nrf_desktop_ble_bond`.
+  * Configuration for ``nrf52833dk_nrf52820`` to allow testing of nRF Desktop dongle on the nRF52820 SoC.
+  * Configuration for `Works With ChromeBook (WWCB)`_.
+  * New documentation :ref:`nrf_desktop_hid_state_pm`.
+
+* Updated:
+
+  * Fn key related macros by moving them to an application specific header file (:file:`configuration/common/fn_key_id.h`).
+  * :ref:`nrf_desktop_buttons` by adding a description about Key ID.
+  * Config channel to no longer use orphaned sections to store module Id information.
+    Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+  * Bluetooth transmission power level for the :ref:`nrf_desktop_ble_adv`.
+    It is now assumed to be 0 dBm during advertising.
+
+  * Documentation and diagrams for the :ref:`nrf_desktop_ble_bond`.
+  * Architecture diagrams.
+  * Documentation with information about fwupd support.
+
+nRF Pelion client
+-----------------
+
+* Removed the nRF Pelion client application as part of removing the Pelion DM support.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Updated:
+
+  * :ref:`slm_description` as part of adding the overlay for Thingy:91 target:
+
+    * Added:
+
+      * :ref:`slm_testing_twi` in :ref:`slm_testing`.
+      * :ref:`slm_connecting_thingy91`.
+
+    * Updated:
+
+      * :ref:`SLM_AT_TWI`.
+      * Additional configuration section by adding information about the configuration files for Thingy:91.
+
+  * SLM UART #XSLMUART section in :ref:`SLM_AT_gen`.
+  * :ref:`slm_config` with the configuration option related to SLM UART.
+
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * :ref:`multiple_adv_sets` sample.
+  * :ref:`ble_nrf_dm` sample.
+
+* Updated:
+
+  * :ref:`peripheral_rscs` sample:
+
+    * Corrected the number of bytes for setting the Total Distance Value and specified the data units. See :ref:`peripheral_rscs_testing` for more details.
+
+  * :ref:`direct_test_mode` sample:
+
+    * Added support for front-end module devices that support 2-pin PA/LNA interface with additional support for the Skyworks SKY66114-11 and the Skyworks SKY66403-11. See the Skyworks front-end module and Vendor-specific packet payload sections in the sample documentation.
+
+  * :ref:`peripheral_hids_mouse` and :ref:`peripheral_hids_keyboard` samples:
+
+    * Added a notice about encryption requirement.
+
+  * :ref:`central_and_peripheral_hrs` sample:
+
+    * Clarified the Requirements section and fixed a link in the Testing section of the sample documentation.
+
+  * :ref:`peripheral_uart` sample:
+
+    * Increased the system workqueue stack size for the :file:`prj_minimal.conf` file.
+
```

Comment:
Please add a following statement (for https://github.com/nrfconnect/sdk-nrf/pull/6794):
```suggestion
* Aligned all samples to use the Bluetooth helper macros: :c:macro:`BT_UUID_16_ENCODE` and :c:macro:`BT_CONN_CB_DEFINE`.
```

Reply:
Added